stetho/src/main/java/com/facebook/stetho/common/android/ViewUtil.java

```diff
@@ -14,6 +19,85 @@ private ViewUtil() {
   }
 
   @Nullable
+  public static View hitTest(View view, float x, float y) {
+    return hitTest(view, x, y, null);
+  }
+
+  // x,y are in view's local coordinate space
```

Comment:
Yes. I'm using the terminology used within View/ViewGroup. Easy enough to clarify locally too.